lib/components/common/tweet/widgets/tweet/content/tweet_actions_bottom_sheet.dart

```diff
@@ -90,7 +90,9 @@ void showTweetActionsBottomSheet(
         title: const Text('copy tweet text'),
         enabled: bloc.tweet.hasText,
         onTap: () {
-          Clipboard.setData(ClipboardData(text: tweet.visibleText));
+          Clipboard.setData(
+            ClipboardData(text: parseHtmlEntities(tweet.visibleText)),
```

Comment:
Instead of parsing the html entities here, It would be better to do it in `_visibleText` in `tweet_data.dart` instead

Reply:
hmmm, looks better! I've made this change!